rasa_core/processor.py

```diff
@@ -286,11 +286,33 @@ def _run_action(self, action, tracker, dispatcher):
                          "code.".format(action.name()), )
             logger.error(e, exc_info=True)
             events = []
+        self._log_bot_utterances_on_tracker(tracker, dispatcher)
         self._log_action_on_tracker(tracker, action.name(), events)
         self._schedule_reminders(events, dispatcher)
 
         return self._should_predict_another_action(action.name(), events)
 
+    def _log_bot_utterances_on_tracker(self, tracker, dispatcher):
+        # type: (DialogueStateTracker, Dispatcher) -> None
+
+        if dispatcher.latest_bot_messages:
+            text = []
```

Comment:
let's just log the messages separately (so for each ebtry ub `dispatcher.katest_bot_messages` one event `BotUttered`)  - or is there a reason not to do that?